.vmtest.yaml

```diff
@@ -6,7 +6,7 @@ amd64:
       qemu: "/zqemu"
 
   VMTEST_KERNEL:
-    container: "ghcr.io/u-root/u-root/test-kernel-amd64@sha256:10ea580ef29468f6d6f4674279586d37e118b145564fd1e99b708370262961a4"
+    container: "ghcr.io/leongross/vmtest/kernel-amd64:main"
```

Comment:
The images were automatically built when you merged the other PR: https://github.com/u-root/u-root/actions/runs/9433728179

if you look here: https://github.com/orgs/u-root/packages?repo_name=u-root

you'll find the new hashes to put in this file

Reply:
Updated the hashes.